### PR TITLE
Added further notes about DLP as a result of the OSFF rountable

### PR DESCRIPTION
**Pull request by robmoffat**
Added ideas covered in the OSFF roundtable, for us to review and discuss.

### CODE REVIEW

File: docs/operations/project-access.md [hunk 1]
```diff
@@ -3,44 +3,136 @@ id: project-access
 title: Working with DLP Limitations to Open Source Access
 sidebar_label: DLP and Open Source
 ---
-Companies in financial services and other highly regulated industries often face difficulties accessing open source projects. The websites or online services that host these projects may be entirely disallowed by the firm's [Data Leak Prevention](https://en.wikipedia.org/wiki/Data_loss_prevention_software) (DLP) software. Alternatively, these resources may be available in a read-only form, preventing contributions or participation in online discussions. 
+Companies in financial services and other highly regulated industries often face difficulties accessing and contributing to open source projects. The websites or online services that host these projects may be entirely disallowed by the firm's [Data Leak Prevention](https://en.wikipedia.org/wiki/Data_loss_prevention_software) (DLP) software, which runs counter to consuming and contributing to projects hosted on public websites such as GitHub and GitLab.
```
Comment (mark-hoare-db) [inline docs/operations/project-access.md:6]: Data Loss Prevention rather than Data Leak Prevention

File: docs/operations/project-access.md [hunk 2]
```diff
@@ -3,44 +3,136 @@ id: project-access
...
 
-## Requestion an exception
+## The Problem
 
-In situations like this, it's common for the IT and/or security teams to provide an exception mechanism. Check to see whether this is possible in your firm.
+Financial institutions run the risk of accidental or deliberate data exfiltration via public websites.  For example, an employee could use [Google Drive](https://drive.google.com) to upload a list of account holders, or leak internal software onto a public archive such as [GitHub](https://github.com) or [DropBox](https://dropbox.com).
```
Comment (mark-hoare-db) [inline docs/operations/project-access.md:10]: The risk is not just exfiltration of data which implies transfer of aggregated data, there is also the risk of inadvertent leakage of sensitive information embedded in code (eg secrets/passwords, network names/topology information etc)

File: docs/operations/project-access.md (hunk 1 above)
Comment (robmoffat) [reply docs/operations/project-access.md]: updated

File: docs/operations/project-access.md (hunk 2 above)
Comment (robmoffat) [reply docs/operations/project-access.md]: I've added that as a second example.  thanks!

**Review by psmulovics — APPROVED** [review]

**Pull request merged**